Src/BootCamp.Chapter/Computer/Cpu.cs

```diff
@@ -2,5 +2,11 @@
 {
     public class Cpu
     {
+        private readonly string _cpu;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_cpu' or refactor the code to use its value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5124353)